src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1138,7 +1138,7 @@
     <value>Partial declarations of '{0}' have conflicting accessibility modifiers</value>
   </data>
   <data name="ERR_PartialMultipleBases" xml:space="preserve">
-    <value>Partial declarations of '{0}' must not specify different base classes</value>
+    <value>Partial declarations of '{0}' must not specify different base types</value>
```

Comment:
>types [](start = 73, length = 5)

I do not think we should make this change. A record class is a class and this error isn't about an interface in the base list, but specifically about the class type.